packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1188,9 +1188,15 @@ export const components = (
             },
         }),
         label: {
+          display: "inline-flex",
+
```

Comment:
Could we avoid the specific margins by using `gap` here? I can't remember why I didn't do that initially.

Reply:
Good call, we can totally do that 🙌